flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -888,29 +873,6 @@ private void tryShutdownTimerService() {
 		}
 	}
 
-	private void checkpointState(
-			CheckpointMetaData checkpointMetaData,
-			CheckpointOptions checkpointOptions,
-			CheckpointMetrics checkpointMetrics) throws Exception {
-
-		CheckpointStreamFactory storage = checkpointStorage.resolveCheckpointStorageLocation(
-				checkpointMetaData.getCheckpointId(),
-				checkpointOptions.getTargetLocation());
-
-		CheckpointingOperation.execute(
```

Comment:
`CheckpointingOperation` should be also be removed in the commit of `[FLINK-16744][task][refactor] extract SubtaskCheckpointCoordinator`